### ISSUE TITLE
Selfee.yml cannot create environment due to package conflicts

### ISSUE THREAD

**Issue by gomj-aba** [EBGU/Selfee#2]
I am trying to use Selfee to track spiders in some videos of spider communal hunting. 
I followed the instructions in the Readme section and have been since trying to create the Selfee environment with the Selfee.yml file.

since my conda base environment uses python=3.9.12, I set up this new environment testSelfee_env with python=3.8.8
(as the YAML file asks for, check code below)

conda create --name testSelfee_env python=3.8.8
source activate testSelfee_env

testSelfee_env has python=3.8.8 now!

So I go ahead and install the dependencies for pyhsmm:
pip install -U future 
pip install requests
pip install pybasicbayes==0.2.2

all fine, running:
pip install pyhsmm

I get error: metadata generation failed
note: This error originates from a subprocess, and is likely not a problem with pip

(do not know if this is the source of the problem, I expect not 
Moving on to creating the env 'Selfee' with the YAML file
expectation: env runs on python=3.8.8.
so no package conflict should occur)

conda env create -f Selfee.yml -n Selfee
(because the environment could not be identified without '-n Selfee', gives error prefix already exists!)

And yet for 1 hour it is checking all sorts of package conflicts!
[selfee package conflict.odg](https://github.com/EBGU/Selfee/files/9811043/selfee.package.conflict.odg)

Any hints are highly appreciated! (new to conda environments so if you think it is a bug please provide some tips to fix that too!)

**Comment by EBGU** [comment]
Hi!

Thank you for your interest in my project.

First of all, I recommend you install dependencies automatedly with: conda env update -f whatever.yml
Second, as I write in the README, you could install pyhsmm from the source (but not pip). "To install manually from the git repo, you'll need cython. Then try this: python setup.py install"
Finally, Selfee is not trained with spiders, and I did not implement any functions for object detection. For spider tracking, you might need YOLO or something similar. If your video has a clean background, you can also modify my RodentTracker, which employing classical computer vision methods.

Good luck with your experiment!

**Comment by gomj-aba** [comment]
Yes! I realized why pip did not work-  thank you for the explanation, will
try it out and get back to you
About the clear background, all the videos have white screen in the
background with spiders on their web in front of it- do you think the
RodentTracker will work (with modifications of course)?
Also a few videos have blurred objects, do you think RodentTracker can deal
with it? Any particular modification you would suggest then?

**Comment by EBGU** [comment]
RodentTracker would work on high-contrast non-overlapping images. RodentTracker only displays one object per frame, so you need to change that. If it cannot work, you can always rely on YOLO :)
